Rakefile

```diff
@@ -109,7 +109,8 @@ DEPENDENCIES = Hash.new([]).merge!(
   '2.3.0/array/bsearch_index' => ['backports/2.3.0/array/dig', 'backports/2.3.0/hash/dig'],
   '2.3.0/array/dig' => ['backports/2.3.0/hash/dig', 'backports/2.3.0/struct/dig'],
   '2.3.0/hash/dig' => ['backports/2.3.0/array/dig', 'backports/2.3.0/struct/dig'],
-  '2.3.0/struct/dig' => ['backports/2.3.0/array/dig', 'backports/2.3.0/hash/dig']
+  '2.3.0/struct/dig' => ['backports/2.3.0/array/dig', 'backports/2.3.0/hash/dig'],
+  '2.6.0/method/compose' => 'backports/2.6.0/proc/compose'
```

Comment:
This is only meant to be used when the specs themselves are dependent on other backports, but not the backports. Here I believe that `method/compose` depends on `proc/compose` and thus *must* require it.